docker-compose-dev.yml

```diff
@@ -1,136 +1,79 @@
-      - ./mediawiki/extensions/JsonConfig:/var/www/html/extensions/JsonConfig
-      - ./mediawiki/extensions/TemplateStyles:/var/www/html/extensions/TemplateStyles
-      - ./mediawiki/extensions/Math:/var/www/html/extensions/Math
-      - ./mediawiki/extensions/MathSearch:/var/www/html/extensions/MathSearch
-      - quickstatements-data:/quickstatements/data
-      - ./mediawiki/extra-install.sh:/extra-install.sh
-    networks:
-      default:
-        aliases:
-         - ${WIKIBASE_HOST}
-         - wikibase-docker.svc
+    restart: 'no'
     environment:
       <<: *wikibase_variables
-      WIKIBASE_PINGBACK:
-      MW_WG_ENABLE_UPLOADS:
 
   wikibase_jobrunner:
-    image: "${WIKIBASE_BUNDLE_IMAGE_NAME}"
-    entrypoint: /bin/bash
-    command:  /jobrunner-entrypoint.sh
-    links:
-      - mysql
-    depends_on:
-      - mysql
-    volumes:
-      - shared:/shared/:ro
-      - ./mediawiki/jobrunner-entrypoint.sh:/jobrunner-entrypoint.sh
-    networks:
-      default:
-        aliases:
-          - wikibase-jobrunner.svc
-    environment:
-      <<: *wikibase_variables
-      MAX_JOBS: ${MAX_JOBS}
-      <<: *wikibase_extra_variables
+    restart: 'no'
 
   mysql:
-    image: "${MYSQL_IMAGE_NAME}"
-    container_name: mardi-mysql
-    volumes:
-      - mediawiki-mysql-data:/var/lib/mysql
+    restart: 'no'
+
+  wdqs-frontend:
+    ports:
+     - "${WDQS_FRONTEND_PORT}:80"
+    restart: 'no'
+
+  quickstatements:
+    ports:
+      - "${QUICKSTATEMENTS_PORT}:80"
+    restart: 'no'
     environment:
-      # CONFIG - Change the default values below (should match values passed to wikibase)
-      MYSQL_DATABASE: ${DB_NAME}
-      MYSQL_USER: ${DB_USER}
-      MYSQL_PASSWORD: ${DB_PASS}
-      MYSQL_RANDOM_ROOT_PASSWORD: 'yes'
-    networks:
-      default:
-        aliases:
-         - mysql.svc
+      <<: *wikibase_variables
 
   backup:
-    image: ${BACKUP_IMAGE_NAME:-ghcr.io/mardi4nfdi/docker-backup:main}
-    links:
-      - mysql
-    depends_on:
-      - mysql
+    restart: 'no'
     volumes:
-      - ${BACKUP_DIR:-./backup}:/data
-    environment:
-      DB_HOST: mysql.svc # internal docker hostname (alias) of the database service
-      DB_NAME: ${DB_NAME}
-      DB_USER: ${DB_USER}
-      DB_PASS: ${DB_PASS}
-      BACKUP_SCHEDULE: ${BACKUP_SCHEDULE:-15 5 * * *}
+      # only for running tests
+      - ./test:/test
 
   reverse-proxy:
-    image: traefik:v2.5
-    ports:
-      - 443:443 # HTTPS port
-      - 80:80 # HTTPS port
-    volumes:
-      - /var/run/docker.sock:/var/run/docker.sock # So that Traefik can listen to the Docker events
-      - ./traefik/traefik.yml:/etc/traefik/traefik.yml:ro
-      - ./traefik/conf/:/traefik-conf/:ro
-      - traefik-letsencrypt:/letsencrypt # Persistent file for ACME Setup (Certificate Store)
-      - traefik-log:/data/log # Persistent file for logging
-    networks:
-     - default
-     - wdqs
-    labels:
-      - traefik.enable=false
+    restart: 'no'
```

Comment:
it might make sense to have a env variable for the restart property, since it has to be set explicitly in for each container.

Reply:
Good idea. Added this to docker-compose-dev